src/contrib/mlir/backend/pass/affine_lowerer.cpp

```diff
@@ -2588,6 +2595,11 @@ namespace
                 Value zero = createZeroConstant(elemTy);
                 iRes(ivs) = zero - val;
             }
+            else if (isa<NGAbsOp>(op))
+            {
+                Value zero = createZeroConstant(elemTy);
+                iRes(ivs) = std_select(lt(val, zero, is_signed(ngTensorType)), zero - val, val);
```

Comment:
#4909 introduces API change for `lt`.

Reply:
Fixed, and I added a unit the value of -0 to the Abs test vector just in case. Is -0 lt zero? Yes it is which is kind of odd.